src/components/course/data/hooks.jsx

```diff
@@ -186,14 +186,12 @@ export function useCourseRunWeeksToComplete(courseRun) {
   const [label, setLabel] = useState(undefined);
 
   useEffect(() => {
-    // this conditional will always fail at courseRun.weeksToComplete === 0, because it always equals false
-    if (courseRun && courseRun.weeksToComplete) {
+    if (courseRun && (courseRun.weeksToComplete || courseRun.weeksToComplete >= 0)) {
```

Comment:
I would suggest adding a comment back in here to explain why it's being done this way.